gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorTimeBasedJobPropCreator.java

```diff
@@ -131,11 +145,36 @@ private boolean shouldProcessFolder(Path inputFolder, Path outputFolder, DateTim
       LOG.info(String.format("Folder time for %s is %s, later than the latest allowed folder time, %s. Skipping",
           inputFolder, folderTime, latestAllowedFolderTime));
       return false;
-    } else if (folderAlreadyCompacted(outputFolder)) {
-      LOG.info(String.format("Folder %s already compacted. Skipping", outputFolder));
-      return false;
+    } else {
+      return true;
+    }
+  }
+
+  /**
+   * Return job properties for a job to handle the appearance of data within inputFolder which is
+   * more recent than the time of the last compaction.
+   */
+  private State createJobPropsForLateData(Path inputFolder, Path outputFolder, Path tempFolder,
```

Comment:
Please use the same parameter names as `createJobProps`, i.e., jobInputDir, jobOutputDir, jobTmpDir